src/external/dftk_to_wannier90.jl

```diff
@@ -0,0 +1,605 @@
+                           bands_plot = false,
+                           num_iter = 500,
+                           num_print_cycles = 50
+                           )
+    
+    ## Check kgrids
+    if (prod(kgrid) != size(scfres.ψ,1))
+        error("The given kgrid doesn't match the one used for the scf calculation")
+    end    
+
+    ## Write in file   
+    open("$prefix.win","w") do f
+
+        write(f,"! $prefix.win file generated by DFTK at "*string(now())*"\n"^2)
+        
+        ## General parameters
+        num_bands = size(scfres.ψ[1],2)
+        write(f,"num_bands "*(" "^9)*"= $num_bands"*"\n")
+        write(f,"num_wann "*(" "^10)*"= $num_wann"*"\n"^2)
+
+        write(f,"num_iter "*(" "^10)*"= $num_iter"*"\n")
+        write(f,"num_print_cycles "*(" "^2)*"= $num_print_cycles"*"\n"^2)
+        
+        ## Messages for the user
+        write(f,"!"^20*" Complete with optional parameters :"*"\n"^2)
+
+
+
+        ## System
+        write(f,"!"^20*" System"*"\n"^2)
+
+        # Unit cell block
+        write(f,"begin unit_cell_cart"*"\n"*"bohr"*"\n")
+        unit_cell = transpose(basis.model.lattice)         #unit cell vectors are in raws in Wannier90.
+        for i in 1:3
+            for j in 1:3
+                write(f,"$(unit_cell[i,j]) ")
+            end
+            write(f,"\n")
+        end
+        write(f,"end unit_cell_cart"*"\n"^2)
+
+        # Atoms bloc
+        atoms = basis.model.atoms
+        write(f,"begin atoms_frac"*"\n")
+        for i in 1:length(atoms)                  #loop over different elements
+            symbol = String(atoms[i][1].symbol)
+            for coord in atoms[i][2]              #loop over atoms of the same element
+                write(f,symbol*"  "*"$(coord[1])  $(coord[2])  $(coord[3])"*"\n")  
+            end
+        end
+        write(f,"end atoms_frac"*"\n"^2)
+
+        ## Projection bloc : message for the user.
+        write(f,"""
+        ! Add a projection bloc only with the option : guess = "win" in dftk2wan_wannierization_files.
+        ! Otherwise, DFTK uses by default gaussian guesses on specified centers 
+        ! or the SCDM method if guess = "SCDM" (not yet implemented).
+
+
+        """)
+
+
+        ## k points
+        write(f,"!"^20*" k_points"*"\n"^2)
+
+        # Bands_plot bloc
+        if bands_plot
+            trash,labels,path = high_symmetry_kpath(basis.model)
+            write(f,"begin kpoint_path"*"\n")
+            for i in 1:size(path[1],1)-1
+                ## write path section A -> B
+                A,B = path[1][i],path[1][i+1]
+                # kpoints are writen in the file with normalized coordinates
+                k_A = round.(DFTK.normalize_kpoint_coordinate.(get!(labels,A,1)), digits = 5)
+                k_B = round.(DFTK.normalize_kpoint_coordinate.(get!(labels,B,1)), digits = 5)
+                # Wannier90 convention for the Γ point is G
+                if A == "\\Gamma"
+                    A = "G"
+                end
```

Comment:
use a function to do this, similar to what we do in detexify_kpoint. Eg you can have a `detexify_kpoint(string, unicode=true)` that returns `Γ` if unicode == true, and `G` if false

Reply:
Hello Michael, the high_symmetry_kpath still gives me "\\Gamma"  labels, so it seems that I didn't update DFTK the right way.
There's something I don't understand because I installed DFTK via the package manager, but we updated the folder I obtained latter via the fork. For me these two things are not the same DFTK so shouldn't I update from pkg directly ? I think I'm missing something here.

Reply:
I'm not exactly sure what you did, but Julia allows you to have multiple versions of the same package. Maybe the (recently updated) docs https://juliamolsim.github.io/DFTK.jl/stable/guide/installation/#Developer-setup help you? You basically have to tell Julia somehow that you want to use the code of your fork. So either you use `julia --project=/some/path/` with the path to your fork or `julia --project` in a shell with working directory `/some/path` or you use the `dev` method outlined in the manual.